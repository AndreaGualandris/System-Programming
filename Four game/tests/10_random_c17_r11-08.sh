#!/bin/bash
exec "$1" 17 11 < tests/10_random_c17_r11-08.txt
