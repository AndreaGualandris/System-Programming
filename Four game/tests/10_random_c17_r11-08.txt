8
5
12
3
16
16
13
2
6
5
13
14
9
6
12
5
10
5
16
7
12
8
6
13
8
13
2
0
1
7
8
8
16
12
5
3
2
16
14
1
3
10
14
3
6
0
11
3
15
16
1
7
12
9
12
13
8
3
2
11
8
1
5
16
9
14
9
3
3
0
8
16
11
11
6
14
13
11
12
16
6
1
11
14
9
5
4
10
15
16
4
1
8
16
2
16
7
6
3
8
1
4
4
11
10
9
10
5
2
9
13
15
1
4
6
1
16
7
13
11
5
8
15
1
12
1
16
16
15
7
10
11
15
15
15
8
8
16
0
7
4
0
0
14
5
9
4
5
1
13
9
10
5
0
9
1
4
14
11
12
14
12
9
4
0
10
10
1
8
0
5
16
4
10
4
13
6
6
5
12
2
6
9
3
3
11
14
