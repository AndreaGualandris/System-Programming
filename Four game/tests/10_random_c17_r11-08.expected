-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
        o
        v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
        o        
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x  o        
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x  o   o    
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x x  o   o    
-----------------
01234567890123456
Player 'o' to move:
                o
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
   x x  o   o   o
-----------------
01234567890123456
Player 'x' to move:
                x
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                x
   x x  o   o   o
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                x
   x x  o   oo  o
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                x
  xx x  o   oo  o
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                x
  xx xo o   oo  o
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x          x
  xx xo o   oo  o
-----------------
01234567890123456
Player 'o' to move:
             o
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x       o  x
  xx xo o   oo  o
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x       o  x
  xx xo o   oox o
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     x       o  x
  xx xo oo  oox o
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     xx      o  x
  xx xo oo  oox o
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
     xx     oo  x
  xx xo oo  oox o
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x           
     xx     oo  x
  xx xo oo  oox o
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
     x           
     xx     oo  x
  xx xo ooo oox o
-----------------
01234567890123456
Player 'x' to move:
     x
     v
-----------------
                 
                 
                 
                 
                 
                 
                 
     x           
     x           
     xx     oo  x
  xx xo ooo oox o
-----------------
01234567890123456
Player 'x' wins!
